src/scripts/settings/SortOptions.ts

```diff
@@ -52,4 +54,16 @@ const SortOptionConfigs: Record<SortOptions, SortOptionConfig> = {
         'getValue': p => p.baseAttack,
         'invert': true,
     },
+
+    [SortOptions.breedingEffiency]: {
+        'text': 'Breeding Efficiency',
+        'getValue': p => (p.baseAttack / pokemonMap[p.name].eggCycles),
+        'invert': true,
+    },
+
+    [SortOptions.highestRegionBreedingEfficiency]: {
+        'text': 'HRBE',
```

Comment:
`HRBE` is all players are going to see in game, it would be ideal if they could know what this is without having to go to discord/wiki/reddit

Maybe just abbreviating some of the words? Something like `Max reg. breeding eff.`?

Reply:
What about `Max Region Breed Eff.` ?

I think the discord uses the term `Breeding Attack`, is that better than `Breeding Efficiency`?

Reply:
Yes, `reg.` doesn't look very clear, might be better to spell that one out entirely

I don't think `Breeding Attack` is a good description, it makes it sound like the pokemon highest on this list will gain the most attack per breed, which is a bit different to what we are actually sorting by.

Might be best to ask people on the discord to see what would make sense without being too long.

Reply:
I might just be misunderstanding this option, But I'm not entirely sure what the use case would be for this one?
Would it not just make more sense to use the `Breeding Efficiency` sort and then filter by the region you want to breed?

Reply:
Well, I'm only in Hoenn so far, and you're right that it's not that important, but even in Hoenn it puts Gyarados second in the list if I want to raise my Hoenn attack. Clefable is next highest out-of-region at around 60, which is kind of low, but still higher than a lot of Hoenn-native pokemon. I think in higher regions, as the penalty for out-of-region pokemon reduces, other pokemon can go higher in the list than region-native pokemon.

I just started playing a few days ago, though, so I can't say for certain how useful it will be.

Reply:
I think it might be best to just do the standard one for now, and we can always re add this further down the line if needed.